### PR TITLE
Fixes #6548: Use the manpath command if available to build a MANPATH in ...

### PR DESCRIPTION
**Pull request by Kegeruneku**
...profile

Ticket: http://www.rudder-project.org/redmine/issues/6548

To be merged in 2.10

### CODE REVIEW

File: rudder-agent/SOURCES/rudder-agent.sh [hunk 1]
```diff
@@ -1,13 +1,26 @@
...
+
+## If $MANPATH is already defined, use it, or build
+## a new one from scratch if it does not
 if [ -z "${MANPATH}" ]
 then
-    MANPATH=/opt/rudder/share/man
+    MANPATH=${MANPATH_COMMAND}/opt/rudder/share/man
 else
-    MANPATH=/opt/rudder/share/man:${MANPATH}
+    MANPATH=${MANPATH}:/opt/rudder/share/man
```
Comment (Kegeruneku) [inline rudder-agent/SOURCES/rudder-agent.sh]: We are not part of the basesystem: our manpages should come last.

Comment (Kegeruneku) [inline rudder-agent/SOURCES/rudder-agent.sh]: (This follows the logic of the PATH in the same file)

File: rudder-agent/SOURCES/rudder-agent.sh [hunk 2]
```diff
@@ -1,13 +1,26 @@
...
 PATH=${PATH}:/var/rudder/cfengine-community/bin
 export PATH
 
-# Add CFEngine manpages to the MANPATH
+# 2 - Build a MANPATH with our manpages in it
+
+## If 'manpath' exists, store its output
+if type manpath >/dev/null 2>&1
+then
+  MANPATH_COMMAND="$(manpath):"
```
Comment (peckpeck) [inline rudder-agent/SOURCES/rudder-agent.sh]: manpath returns an error if MANPATH is not empty

**Comment by peckpeck** [comment]
OK good

**Pull request merged**